### PR TITLE
db: Run missing migration for EventTag index

### PR DESCRIPTION
**Pull request by jan-auer**
Running migrations from a clean master checkout gave me this.

### CODE REVIEW

**Comment by getsentry-danger** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          2 Warnings
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">You should update CHANGES due to the size of this PR</td>
    </tr>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">PR includes migrations</td>
    </tr>
  </tbody>
</table>

## Migration Checklist

- [ ] new columns need to be nullable (unless table is new)
- [ ] migration with any new index needs to be done concurrently
- [ ] data migrations should not be done inside a transaction
- [ ] before merging, check to make sure there aren't conflicting migration ids


<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">danger</a>
</p>

**Review by mattrobenolt — CHANGES_REQUESTED** [review]
Absolutely blocking this regardless since something else is wrong.

**Comment by jan-auer** [comment]
Remove [`eventtag.py:31`](https://github.com/getsentry/sentry/blob/master/src/sentry/tagstore/v2/models/eventtag.py#L31) then, instead?

**Comment by mattrobenolt** [comment]
It's a little bit confusing right now, but those models aren't what the migration isbeing applied to. Here is the commit when we dropped the index: https://github.com/getsentry/sentry/commit/ba30ca558c59cdf53152673ab4746011ff145d2b

**Comment by bretthoerner** [comment]
> Running migrations from a clean master checkout gave me this.

What's the exact command?

```
$  sentry django schemamigration --auto sentry
Nothing seems to have changed.
```

**Comment by jan-auer** [comment]
That's the command I used. Was also able to reproduce on @HazAT's machine

**Comment by mattrobenolt** [comment]
@bretthoerner umm.

```
$ sentry django schemamigration sentry --auto
 + Added index for ['project_id', 'key_id', 'value_id'] on sentry.EventTag
Created 0368_auto__add_index_eventtag_project_id_key_id_value_id.py. You can now apply this migration with: ./manage.py migrate sentry
```

Lemme figure out what went wrong by reading all of this `meta` stuff by hand. I assume there was an out of order migration or something merged in which the frozen state is wrong.

Either way, @jan-auer thanks for bringing this to our attention. But let's close this PR for now since we definitely don't want this.

**Comment by mattrobenolt** [comment]
tl;dr The issue is because of your local pyc files. Delete them. :)

The problem is that `eventtag.py` was moved from `src/sentry/models/eventtag.py` -> `src/sentry/tagstore/legacy/models/eventtag.py` But this left back a pyc at `src/sentry/models/eventtag.pyc` which meant your EventTag model was getting pulled in from that old pyc file which has the wrong model definition.
